packages/grid/src/Grid.tsx

```diff
@@ -355,6 +355,7 @@ class Grid extends PureComponent<GridProps, GridState> {
     this.handleMouseUp = this.handleMouseUp.bind(this);
     this.handleResize = this.handleResize.bind(this);
     this.handleWheel = this.handleWheel.bind(this);
+    this.getSelectedRanges = this.getSelectedRanges.bind(this);
```

Comment:
Binding this here is not strictly necessary, as `getSelectedRanges` isn't passed in anywhere. Should just be able to call `grid.getSelectedRanges` directly.

Reply:
I'm getting `TypeError: Cannot read properties of undefined (reading 'state')` when I'm not binding it. I believe it's because the function is reading `selectedRanges` from `this.state`.